Gruntfile.js

```diff
@@ -1,10 +1,24 @@
 module.exports = function (grunt) {
+	var staticExampleFiles = [ 'src/examples/**', '!src/examples/**/*.js' ];
```

Comment:
what is `examples` ? examples belong in `dojo/examples`, not under `dojo/widgets`